jplag.frontend.cpp/src/main/java/jplag/cpp/Language.java

```diff
@@ -52,15 +52,15 @@ public boolean usesIndex() {
 	}
 
 	public jplag.Structure parse(File dir, String[] files) {
-		return this.scanner.scan(dir, files);
+		return this.scanner.parse(dir, files);
 	}
 
 	public boolean errors() {
 		return this.scanner.getErrors();
 	}
 
 	public int noOfTokens() {
-		return jplag.cpp.CPPToken.numberOfTokens();
+		return scanner.getEndOfFileToken("").numberOfTokens();
```

Comment:
Should we not add a `staticNumberOfTokens` method to the CPPToken class, for symmetry with other languages?